extensions/amp-analytics/amp-analytics.md

```diff
@@ -546,7 +546,9 @@ each relevant trigger for more details.
 The selector properties are:
 
 -   `selector` This property is used to find an element or a collection of
-    elements using CSS/DOM query. The semantics of how the element is matched
+    elements using CSS/DOM query.
+
+- The semantics of how the element is matched
```

Comment:
Revert this change, no need for newlines.

Reply:
Just reverted the changes from amp-analytics.md and package-lock.json. 
Those were changes made by the amp-video multi selector documentation PR that has not been merged yet.